courses/fundamentals_of_ada/060_record_types.rst

```diff
@@ -228,6 +228,33 @@ Assignment Examples
       Phase1.Real := Phase2.Real;
    end;
 
+----------------------
+Constant and Limited
```

Comment:
I think "constant" just confuses the issue. (They _can_ be on the right side, they _can_ be compared).
Just replace the first two bullets with something like "limited adds certain restrictions to record types (and other types described later)"

Reply:
I reworded it, let me know :)